build.sbt

```diff
@@ -41,12 +41,15 @@ val javaWebsocketV = "1.4.0"
 val kindProjectorV = "0.10.3"
 val betterMonadicForV = "0.3.1"
 
+lazy val scalaVersions = scala.io.Source.fromFile("scalaVersions.dhall")
+  .mkString.replaceAll("""\[|\]|"| """, "").trim.split(",")
```

Comment:
`scalaVersions.dhall` is also valid JSON - not sure if that's something we'd want to rely on, though.

Reply:
Yeah, am also not sure about this.

Obviously, it would be best to use a Dhall implementation for parsing, but [dhall-eta](https://github.com/eta-lang/dhall-eta), the only JVM implementation, uses [eta](https://github.com/typelead/eta), which is unmaintained.

Another option would be to move `scalaVersions.dhall` to the `dhall` directory and generate a `scalaVersion.json` from it—this way, we could use all Dhall features in it, and do not rely on it accidentally being JSON. Actually, I think I slightly prefer this option—what do you think?

Reply:
I've toyed with reviving the scala-dhall project, but there's this pesky open source HTTP library that's eating up all of my open source time at the moment.

Reply:
I would be interested in contributing to a scala-dhall, but a fully compliant implementation is indeed a big project, and I don't think it is worth to also reimplement `dhall format` and `dhall-to-yaml` in Scala, even though it is indeed sad to have non-JVM dependencies.